### DISCUSSION TITLE
RPM on OSD

### DISCUSSION THREAD

**Discussion by RobertoD91** [iNavFlight/inav#10375] [Q&A] [answered]
Hi, just to confirm: is it possible to view the RPM of the motors on the OSD?

Configuration:

- INAV 7.1.2
- 2 ESCs with Bluejay 0.21
- DShot 600 enabled
- OSD with WTFOS DJI

Thank you!

**Comment by MrD-RC** [comment, marked as answer]
Yes, it is possible to show ESC telemetry. So long as the ESC has a telemetry output. You need to connect that to an RX on the flight controller.

**Reply by RobertoD91** [reply]
Ok, thank you. So, if I use an ESC with dshot bidirectional telemetry on a single wire, is it not possible?

**Reply by MrD-RC** [reply]
No

**Reply by RobertoD91** [reply]
Ok, thank you!
